client/src/research/BubbleChart.js

```diff
@@ -24,8 +26,30 @@ class BubbleChart extends Component{
     };
     var reactNode = <div style={styles} title={`n=${n}`}></div>;
     return reactNode;
+  } 
+  makeNormBubble(percentage, n){
+    var color = (percentage>0)? "black": "green";
```

Comment:
I think I got mixed on this in person, can you check and clarify the explanation in the pull request?